libs/features/search/src/ui/components/search-bar/search-bar.component.ts

```diff
@@ -1,11 +1,20 @@
-import { Component } from '@angular/core';
+import { Component, Output, EventEmitter} from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
 
 @Component({
    selector: 'lib-search-bar',
    standalone: true,
-   imports: [CommonModule],
+   imports: [CommonModule,
+   FormsModule],
    templateUrl: './search-bar.component.html',
    styleUrls: ['./search-bar.component.scss']
 })
-export class SearchBarComponent {}
+export class SearchBarComponent {
+   query!: string;
```

Comment:
Decorators should be placed before properties. Example: 
`
@Output() queryEmitter = new EventEmitter<string>()
query!: string;
`
This is just to keep things consistent and neat.